app/jobs/media_object_indexing_job.rb

```diff
@@ -15,6 +15,11 @@
 class MediaObjectIndexingJob < ApplicationJob
   queue_as :media_object_indexing
 
+  # Only enqueue one indexing job at a time but once one starts running allow others to be enqueued
+  # because the first step of the job is fetching the object from fedora at which point it might
+  # be stale and subsequent jobs might get a newer version of the media object.
+  unique :until_executing, on_conflict: :log
```

Comment:
Are these built in Ruby/Rails functions? All I'm finding online is references to the "sidekiq-unique-jobs" gem when I try to look up `until_executing`. Maybe I missed it, but I didn't see that gem in the Gemfile.